Navaids/VOR_UK.txt

```diff
@@ -53,23 +53,21 @@ WIK 113.600 N058.27.31.740 W003.06.01.340 ; Wick
 WIN 115.750 N051.47.26.300 W002.03.00.770 ; Winstone (DME)
 
 ;UK TACANs
-BSO 110.000 N051.36.52.711 W001.05.57.522 ; Benson
```

Comment:
Can't directly suggest, but on line 46 check SWB DVOR/DME co-ords, should be N052.47.52.830 W002.39.44.350 per https://www.aidu.mod.uk/aip/pdf/enr/ENR-4-1.pdf

Reply:
CWL and BZ both still exist, and the co-ordinates were correct too so just leave these as was please (CWL defined in https://www.aidu.mod.uk/aip/pdf/ad/EGYD-Cranwell-Textual.pdf and BZ in https://www.aidu.mod.uk/aip/pdf/ad/EGVN-Brize-Norton-Textual.pdf)